README.md

```diff
@@ -9,14 +9,26 @@ It can locate devices/emulators, install a given application, run it and collect
 It handles application crashes by collecting crash dumps and supports different types of connection modes (Network, USB cable).
 It can output test results in various different formats from text to xUnit/NUnit XML.
 
-## Running the tool
+## System requirements
 
 The tool requires **.NET Core 3.1.201** or later to be run. It is packaged as a `dotnet tool` command and can be installed using the [dotnet tool CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/).
 
 - The iOS scenarios require you to run the tool on MacOS with full Xcode installation
-- Android scenarios are supported on Linux, macOS and Windows systems
+- Android scenarios are supported on Linux, macOS and Windows systems with Android emulators
 - Browsers scenarios are supported on Linux systems
 
+## Try the tool out quickly
+
+If you want to test the tool quickly (MacOS and Linux only), following script will install the required .NET SDK and the XHarness tool locally in the current folder.
+It will also pull a sample iOS app for you and you can then try running the XHarness tool right away (instructions will be printed on screen):
```

Comment:
I don't think we should do this in the bootstrap script. It's unrelated to xharness and will only work on macOS.